### PR TITLE
various LaTeX-lab changes

### PR DESCRIPTION
**Pull request by u-fischer**
This contains a number of small changes in latex-lab:

 * language detection now makes use of \BCPdata, 
 * a firstaid to handle https://github.com/latex3/tagging-project/issues/69 has been added
 * "presentation" tables could have unwanted header rows, this has been corrected.
 * the math module now uses tagging sockets.
 * the pdfmanagement has been updated in the texmf
 
Additionally tests of latex-lab firstaid has been activated (that was missing before).

# Internal housekeeping

## Status of pull request
[x] Ready to merge

## Checklist of required changes before merge will be approved
- [x] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `changes.txt` updated
- [ ] Rollback provided (if necessary)? n/a
- [ ] `ltnewsX.tex` (and/or `latexchanges.tex`) updated n/a

### CODE REVIEW

**Review by FrankMittelbach — APPROVED** [review]
a few minor comments

File: required/latex-lab/config-firstaid.lua [hunk 1]
```diff
@@ -6,6 +6,6 @@
 checkengines = {"pdftex"}
 checksearch  = true
 testfiledir  = "testfiles-firstaid"
-
+excludetests={"test-cleveref"} -- doesn't work if firstaid is not in the format
```
[the remark this answers is not in the document]
Comment (u-fischer) [reply required/latex-lab/config-firstaid.lua:9]: no, see my mail on the team list.

File: required/latex-lab/latex-lab-table.dtx [hunk 2]
```diff
@@ -570,7 +573,7 @@
                     \prop_if_exist:cT
                       { g__tag_struct_ \l_@@_tmpa_tl _prop }
                       {
-                        \@@_struct_prop_gput:Vnn \l_@@_tmpa_tl {S}{/Artifact}
+                        \exp_args:No \__tag_struct_prop_gput:nnn {\l_@@_tmpa_tl} {S}{/Artifact}
```
[the remark this answers is not in the document]
Comment (u-fischer) [reply required/latex-lab/latex-lab-table.dtx:576]: yes, but the variant is missing in tagpdf, I will add it there and then correct here.

File: texmf/tex/latex/pdfmanagement-testphase/color-ltx.sty [hunk 3]
```diff
@@ -24,7 +24,7 @@
 %% This file has the LPPL maintenance status "maintained".
 %%
 \NeedsTeXFormat{LaTeX2e}[1995/12/01]
-\ProvidesPackage{color-ltx}[2024-04-09 v0.96h
+\ProvidesPackage{color-ltx}[2024-05-23 v0.96i
```
[the remark this answers is not in the document]
Comment (u-fischer) [reply texmf/tex/latex/pdfmanagement-testphase/color-ltx.sty:27]: in the pdfmanagement all files have the same version number.

File: required/latex-lab/config-firstaid.lua (hunk 1 above)
[the remark this answers is not in the document]
Comment (FrankMittelbach) [reply required/latex-lab/config-firstaid.lua:9]: then I guess that is one for @josephwright

**Pull request merged**